libs/SmartStore/build.gradle

```diff
@@ -10,14 +10,15 @@ apply plugin: 'org.jetbrains.kotlin.android'
 
 dependencies {
     api project(':libs:SalesforceSDK')
+    //noinspection GradleDependency -  Needs to line up with supported sqlcipher version.
     api 'androidx.sqlite:sqlite:2.0.1'
     api 'net.zetetic:android-database-sqlcipher:4.5.2'
-    implementation 'androidx.core:core-ktx:+'
```

Comment:
**This is important.**  Not setting a version for `core-ktx` can cause duplicate headers in consuming apps.